pom.xml

```diff
@@ -98,6 +102,38 @@
                     <autoReleaseAfterClose>true</autoReleaseAfterClose>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>com.googlecode.maven-download-plugin</groupId>
+                <artifactId>download-maven-plugin</artifactId>
+                <version>1.7.1</version>
+                <configuration>
+                    <unpack>false</unpack>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>krausening</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <url>https://repo1.maven.org/maven2/org/technologybrewery/krausening/krausening/maven-metadata.xml</url>
+                            <outputDirectory>${project.build.directory}/krausening.xml</outputDirectory>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>foo</id>
```

Comment:
A: Could use a more descriptive id and output file.

Reply:
Actually, this is was test that I didn't see when I pushed this. Will remove altogether.